modules/claims_api/lib/claims_api/evss_bgs_mapper.rb

```diff
@@ -20,7 +20,7 @@ def map_and_build_object # rubocop:disable Metrics/MethodLength
       claim['data'] = @data
 
       claim['evss_id'] = @data['benefit_claim_id']
-      claim['data']['contention_list'] = @data['contentions'] || []
+      claim['data']['contention_list'] = [@data['contentions']].nil? ? @data['contentions'] : ['']
```

Comment:
It looks like  `claim['data']['contention_list']` would never has a value other then [''] or nil? If I pull that claim from the ticket (600400703 ) from staging I do see values in there `"contention_list": "Generalized anxiety disorder (New), Right shoulder (New), back pain (New)"` which I think would get removed here and get set to [''] here.  I could be missing something though.